dts/bindings/can/microchip,mcp2515.yaml

```diff
@@ -18,6 +18,7 @@ properties:
     int-gpios:
       type: phandle-array
       required: true
+      description: Active low interrupt signal
```

Comment:
The `GPIO_ACTIVE_LOW`, `GPIO_ACTIVE_HIGH` flags are very useful, though their purpose / usage may be a bit confusing. Unfortunately, this comment is incorrect.

In principle the flags are there to encode cause -> effect relationship. This is easy to understand when taking button as an example. Regardless of the way the button is designed and the circuitry surrounding it if we attach `GPIO_ACTIVE_LOW` to the pin it means: if the button is pressed we expect input signal on the pin to be low. If we attach `GPIO_ACTIVE_HIGH`  to the pin it means: if the button is pressed we expect input signal on the pin to be high. The same holds true for MCP2515 chip. The flag is there to tell the driver what happens when MCP2515 asserts its interrupt line low. In a typical case the interrupt output of MCP2515 will be wired directly to the GPIO pin on the MCU. This should be described in DTS by adding `GPIO_ACTIVE_LOW` flag to the pin property. However, it's also legal to attach `GPIO_ACTIVE_HIGH` flag in case the signal from MCP2515 to MCU is routed on the board via an inverter. If we didn't want to allow for this flexibility we could work directly with raw signals.

Reply:
@mnkp what do you propose would be better?

Reply:
There's some information at https://github.com/zephyrproject-rtos/zephyr/issues/20017#issuecomment-549315497 where I have proposed something like this (from #20295) be standardized:
```
    alert-gpios:
￼      type: phandle-array
￼      required: false
￼      description: ALERT pin
￼      description: >
￼        ALERT pin.
￼
￼        This pin signals active high when produced by the sensor.  The
￼        property value should ensure the flags properly describe the
￼        signal that is presented to the driver.
```

Reply:
Thanks for the in-depth comments! I noticed the verbosity in your PR before but didn't fully understand which purpose it served so I felt just stating the 'datasheet' view would be enough. 

I aligned it with the format from @pabigot now.